python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1607,8 +1618,30 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
         badMaskPlanes.append("CLIPPED")
         badPixelMask = afwImage.Mask.getPlaneBitMask(badMaskPlanes)
 
-        return AssembleCoaddTask.assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
-                                          spanSetMaskList, mask=badPixelMask)
+        retStruct = AssembleCoaddTask.assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList,
+                                               spanSetMaskList, mask=badPixelMask)
+
+        # Propagate PSF-matched EDGE pixels to coadd SENSOR_EDGE and INEXACT_PSF
+        # Psf-Matching moves the real edge inwards
+        self.applyAltEdgeMask(retStruct.coaddExposure.maskedImage.mask, spanSetMaskList)
+        return retStruct
+
+    def applyAltEdgeMask(self, mask, altMaskList):
+        """!
+        \brief Propagate alt EDGE mask to SENSOR_EDGE AND INEXACT_PSF planes
```

Comment:
Guessing you're only using doxygen to keep things consistent, right? Otherwise, numpydoc.

Reply:
I'd rather keep the whole file consistent. I just filed a ticket DM-13587